### PR TITLE
Bug 1217033 - Make pinned pages accessible on homescreen. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7226972bbaceb96091c73757c6be7cbd0302100f

File: apps/homescreen/js/pages.js [hunk 1]
```diff
@@ -153,33 +158,40 @@ const PAGES_ICON_SIZE = 30;
...
+        name: card.title,
+        icon: card.icon,
+        remote: true
+      };
+
+      window.open(card.dataset.id, '_blank', Object.keys(features).
+        map(function eachFeature(key) {
+          return encodeURIComponent(key) + '=' +
+            encodeURIComponent(features[key]);
+        }).join(','));
```
Comment (gmarty) [inline apps/homescreen/js/pages.js]: Can you use arrow functions here for better readability?
```js
Object.keys(features).
        map(key => encodeURIComponent(key) + '=' + encodeURIComponent(features[key]))
```

Comment (Cwiiis) [inline apps/homescreen/js/pages.js]: Good idea!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1efede9b04ce8a99683172dfcfb327f33a460f56

**Pull request merged**